apps/public-ui/pages/sitemap.xml.tsx

```diff
@@ -0,0 +1,46 @@
+const Sitemap = () => {
+  return null;
+};
+
+export const getServerSideProps = async ({ res }) => {
+  const BASE_URL = 'https://ezsailingcharters.com';
+
+  // Only added base URL and paths with content
+  const paths = [
+    '',
+    'charters',
+    'special-occasions',
+    'story',
+    'faq',
+    'catering',
+  ];
+
+  const urls = paths.map(
+    (path) =>
+      `<url>
+        <loc>${BASE_URL}/${path}</loc>
+        <lastmod>${new Date().toISOString()}</lastmod>
```

Comment:
Does Google care if lastmod changes more frequently than the page does? As is, lastmod will always be the current date time.